spec/support/factory_bot.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
```

Comment:
This config means FactoryBot's methods like `create` are available in all specs without prefixing with `FactoryBot`, for example:

```diff
-page = FactoryBot.create :top_level_page
+page = create :top_level_page
```

Reply:
Apparently I realised that halfway through the spec file and didn't go back to fix the earlier ones. How odd. Cheers! :)